### ISSUE TITLE
Not installable with Laravel 9.38.0

### ISSUE THREAD

**Issue by syntafin** [atymic/twitter#402]
Tried to install it under Laravel 9.38.0 but got the following error message:

```
➜  DotSexyV4 git:(dev) ✗ composer require atymic/twitter
Using version ^3.1 for atymic/twitter
./composer.json has been updated
Running composer update atymic/twitter
Gathering patches for root package.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - atymic/twitter 3.1.15 requires php-di/php-di ^6.2 -> satisfiable by php-di/php-di[6.2.0, ..., 6.4.0].
    - atymic/twitter[3.1.0, ..., 3.1.14] require illuminate/support 6 - 8 -> found illuminate/support[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - php-di/php-di[6.2.0, ..., 6.4.0] require psr/container ^1.0 -> found psr/container[1.0.0, ..., 1.x-dev] but the package is fixed to 2.0.2 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires atymic/twitter ^3.1 -> satisfiable by atymic/twitter[3.1.0, ..., 3.1.15].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require atymic/twitter:*" to figure out if any version is installable, or "composer require atymic/twitter:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

Enviroment Information:
* Laravel Version 9.38.0
* PHP Version 8.1.11
* Composer Version 2.3.10

**Comment by a-j-n** [comment]
try to install it with -W 
`composer require atymic/twitter -W`

**Comment by CyberPunkCodes** [comment]
> try to install it with -W `composer require atymic/twitter -W`

It might work, but it isn't a real solution. It downgrades two important packages by a whole version:

```bash
- Downgrading psr/container (2.0.2 => 1.1.2)
- Downgrading symfony/service-contracts (v3.2.0 => v2.5.2)
```

Ran on a brand new Laravel 9.45.1

**Comment by atymic** [comment]
Closed by #411
